client/js/components/statement/assessmentTable/AssessmentTableFilter.vue

```diff
@@ -61,7 +61,8 @@
       class="c-at__controls o-sticky o-sticky--border space-stack-xs u-pt-0_5 u-pb-0_25"
       ref="header">
       <search-and-sorting
-        :search-term="searchTerm" />
+        :search-term="searchTerm"
+        @exportModal:toggle="tab => $emit('exportModal:toggle', tab)" />
```

Comment:
has the change todo with fixing the tests ?

Reply:
Yes, I removed the `$root.$on` in ExportModal.vue and replaced it with normal events, so now we need to bubble them up through the component chain